### ISSUE TITLE
Installing packages that require building C code with the builtin interpreter fails

### ISSUE THREAD

**Issue by mmomtchev** [mmomtchev/pymport#40]
When using the built-in interpreter, the reported directory for the include files is not correct:

```
  × Building wheel for xxhash (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [18 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-cpython-311
      creating build/lib.linux-x86_64-cpython-311/xxhash
      copying xxhash/__init__.py -> build/lib.linux-x86_64-cpython-311/xxhash
      copying xxhash/version.py -> build/lib.linux-x86_64-cpython-311/xxhash
      copying xxhash/py.typed -> build/lib.linux-x86_64-cpython-311/xxhash
      copying xxhash/__init__.pyi -> build/lib.linux-x86_64-cpython-311/xxhash
      running build_ext
      building '_xxhash' extension
      creating build/temp.linux-x86_64-cpython-311
      creating build/temp.linux-x86_64-cpython-311/deps
      creating build/temp.linux-x86_64-cpython-311/deps/xxhash
      creating build/temp.linux-x86_64-cpython-311/src
      gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -Ideps/xxhash -I/home/runner/work/pymport/pymport/lib/binding/linux-x64/include/python3.11 -c deps/xxhash/xxhash.c -o build/temp.linux-x86_64-cpython-311/deps/xxhash/xxhash.o
      error: command 'gcc' failed: No such file or directory
      [end of output]
```

**Comment by mmomtchev** [comment]
Normally, the built-in interpreter is targeting installations where compilation is impossible.

If rebuilding Python packages from C/C++ sources, the best solution is to also rebuild `pymport` itself and to use a shared Python environment.

`npm install pymport --build-from-source`

**Comment by mmomtchev** [comment]
setuptools have been upgraded for everyone

**Comment by mmomtchev** [comment]
Problem solved by `pympip install setuptools==65.1.1`

**Comment by mmomtchev** [comment]
Issue still under discussion in pipa: https://github.com/pypa/distutils/issues/178

**Comment by mmomtchev** [comment]
McGyver fix in 1.4.0-rc1 (#65)
